<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FRX Labortnetzteil</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressourcen.qrc">
    <normaloff>:/icon_250.ico</normaloff>:/icon_250.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cbserial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>921</width>
      <height>361</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Control unit</string>
     </attribute>
     <widget class="QLCDNumber" name="lcdVolt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>111</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Outline</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdCurrent">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>241</width>
        <height>111</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Outline</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblV">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>58</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblA">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>58</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QDial" name="dial_v">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
     </widget>
     <widget class="QDial" name="dial_a">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_out">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>OUT ON</string>
      </property>
     </widget>
     <widget class="QChartView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>471</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Serial</string>
     </attribute>
     <widget class="QLineEdit" name="txtcmd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtlog">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>3</y>
        <width>761</width>
        <height>261</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="overwriteMode">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>289</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bnt_err">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>10</y>
        <width>90</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ERR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bnt_failt">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>40</y>
        <width>90</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>FAILD</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>information</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>AR488 Version:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Device:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ROM:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Max Volt:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Max. Ampere:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ar488_ver">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>741</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_device_Ver">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>751</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_rom_version">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>751</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_volt">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>761</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_ampere">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>761</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="bnt_SerialDevices">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressourcen.qrc"/>
 </resources>
 <connections/>
</ui>
